src/simd_128.h

```diff
@@ -81,6 +81,10 @@ inline uint16_t TESTZ(VecType x, VecType y)
 {
     return _mm_testz_si128(x, y);
 }
+inline int is_all_zeros(VecType x)
```

Comment:
This should return a `bool` since it's a predicate (starts with `is_`).

Reply:
yes, it's better

- [ ] `is_xx` return bool